ios/FluentUI/Tooltip/Tooltip.swift

```diff
@@ -63,29 +77,77 @@ open class Tooltip: NSObject {
     private var gestureView: UIView?
     private var dismissMode: DismissMode = .tapAnywhere
 
-    private override init() { }
+    @objc public init() {
+        super.init(frame: .zero)
+
+        // Update appearance whenever `tokenSet` changes.
+        tokenSetSink = tokenSet.sinkChanges { [weak self] in
+            self?.updateAppearance()
+        }
+    }
+
+    required public init?(coder aDecoder: NSCoder) {
+        preconditionFailure("init(coder:) has not been implemented")
+    }
 
     /// Displays a tooltip based on the current settings, pointing to the supplied anchorView.
     /// If another tooltip view is already showing, it will be dismissed and the new tooltip will be shown.
     ///
     /// - Parameters:
     ///   - message: The text to be displayed on the new tooltip view.
+    ///   - title: The optional bolded text to be displayed above the message on the new tooltip view.
     ///   - anchorView: The view to point to with the new tooltip's arrow.
     ///   - preferredArrowDirection: The preferrred direction for the tooltip's arrow. Only the arrow's axis is guaranteed; the direction may be changed based on available space between the anchorView and the screen's margins. Defaults to down.
     ///   - offset: An offset from the tooltip's default position.
     ///   - screenMargins: The margins from the window's safe area insets used for laying out the tooltip. Defaults to 16.0 pts on all sides.
     ///   - dismissMode: The mode of tooltip dismissal. Defaults to tapping anywhere.
     ///   - onTap: An optional closure used to do work after the user taps
-    @objc public func show(with message: String, for anchorView: UIView, preferredArrowDirection: ArrowDirection = .down, offset: CGPoint = CGPoint(x: 0, y: 0), screenMargins: UIEdgeInsets = defaultScreenMargins, dismissOn dismissMode: DismissMode = .tapAnywhere, onTap: (() -> Void)? = nil) {
+    @objc public func show(with message: String,
```

Comment:
to avoid bunch of objc clients to be re-written we should have a separate convenience metod that has title param and leave this as is.

Reply:
but if the title param is given a default value of nil, the existing callers of the show method don't need to be changed, right?

Reply:
swift works that way. not Objc

Reply:
I tried to do this, but am crashing when not passing in a title because it can't tell which method I'm trying to use. Should I just give the function a different name?